ImageLounge/src/DkGui/DkNoMacs.cpp

```diff
@@ -211,6 +211,7 @@ void DkNoMacs::init()
 
     // connections to the image loader
     connect(getTabWidget(), &DkCentralWidget::imageUpdatedSignal, this, QOverload<QSharedPointer<DkImageContainerT>>::of(&DkNoMacs::setWindowTitle));
+    connect(getTabWidget(), &DkCentralWidget::thumbViewLoadedSignal, this, &DkNoMacs::setWindowTitleDirectory);
```

Comment:
```suggestion
    connect(getTabWidget(), &DkCentralWidget::thumbViewLoadedSignal, this, [this](const QString &path) {
        setWindowTitle(path);
    });
```

You can do this because `DkNoMacs::setWindowTitleDirectory` has default values for the last three parameters.

Reply:
done, thanks